packages/astrocite-bibtex/src/__tests__/cases/tilde_comments_underscores_diacritics_sup_sub_nodoti_math.bib

```diff
@@ -0,0 +1,15 @@
+@comment{jabref-meta: groupsversion:3;}
+
+@STRING{ H&R       = "Harper \& Row, Publishers"}
+
+@Article{nau-etal:2001_a,
+	AUTHOR = {Fausto Giunchiglia and Paolo Traverso},
+	TITLE = "Planning as Model-checking",
+	BOOKTITLE = {Pro{\c c}eedings of {ECP-99}},
+	YEAR = {1999},
+	PUBLISHER = SPRINGER,
+	NOTE = {Paper accompanying $^i_n$v{\'\i}ted talk 
```

Comment:
Looks like there is indeed a break in the parser happening here: 

https://github.com/dsifford/astrocite/blob/fd9d89f607adbea9846433f4474bd2500564896e/packages/astrocite-bibtex/src/parser.ts#L88-L94

Can you look into this and see why this might be happening. It looks like it's not being parsed to a string, but rather an object (hence the `[Object object]`)

Reply:
Got it, test asjusted, run green now